CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ## 🧰 Bug fixes 🧰
 
+- `jaeger` receiver/exporter: Parse/set Jaeger status as OTel spec defines (#6682)
```

Comment:
I agree it's more like a bug fix, but can be a breaking change for those who rely on the OpenCensus style status codes. Should we move it to the _Breaking changes_ section?

Reply:
sure thing